types/react-i18next/index.d.ts

```diff
@@ -1,24 +1,40 @@
-// Type definitions for react-i18next 4.6
+// Type definitions for react-i18next 7.0
```

Comment:
can we bump to exact version of the library as well ( `7.3.` for this updated typed API )

Reply:
specified 7.3 to make it clearer.